doc/performance.md

```diff
@@ -82,11 +83,12 @@ without bound:
 
   ```c++
   dom::parser parser(0); // This parser will refuse to automatically grow capacity
-  simdjson::error_code allocate_error = parser.allocate(1000*1000); // This allocates enough capacity to handle documents <= 1MB
-  if (allocate_error) { cerr << allocate_error << endl; exit(1); }
+  auto error = parser.allocate(1000*1000); // This allocates enough capacity to handle documents <= 1MB
+  if (error) { cerr << error << endl; exit(1); }
```

Comment:
(I wonder whether exit(error) would ever make sense?

Reply:
(Minor point, more a note for myself.)

Reply:
It could. I've learned to be careful with overallocating exit codes early, though. People end up relying on them in ways that constrain future changes :)